mos-platform/mega65/crt/divmod.h

```diff
@@ -0,0 +1,244 @@
+#ifndef __SLOW_DIV
+
+#define DIVBUSY 0xD70F
+#define DIVOUT_FRAC 0xD768
+#define DIVOUT_WHOLE 0xD76C
+#define MULTINA 0xD770
```

Comment:
Since we're in C++, `static auto &MULTINA = *(volatile unsigned long *)(0xD770);`, that should help make the results more readable.